### PR TITLE
Update React apps to Next 9

### PR DESCRIPTION
**Pull request by cnasc**
# Description

This PR updates all the React apps to use Next 9. Most of my changes were as found here: https://github.com/zeit/next.js/blob/canary/UPGRADING.md

Some tests failed. In general they were simple fixes. More tests failed in the paywall, but all of the failing tests were for dead code so I just deleted the code and the tests.

# Issues

Fixes #4814 

# Checklist:

- [ ] 1 PR, 1 purpose: my Pull Request applies to a single purpose
  - [ ] This PR only contains configuration changes (package.json, etc.)
  - [ ] This PR only contains code changes (if configuration changes are required, do a separate PR first, then re-base)
- [x] My code follows the style guidelines of this project, including naming conventions
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] If my code adds or changes components, I have written corresponding stories with Storybook
- [x] I have performed a self-review of my own code
- [ ] If my code involves visual changes, I am adding applicable screenshots to this thread

### CODE REVIEW

**Comment by cnasc** [comment]
Hmm, these all run fine locally. I'll pick this up later and see what's up.

**Comment by cnasc** [comment]
The problem is webpack, but not sure how to resolve

File: unlock-app/src/pages/_error.js [hunk 1]
```diff
@@ -1,34 +1,18 @@
...
 import withConfig from '../utils/withConfig'
 
 class Error extends React.Component {
-  static getInitialProps({ res, err, req }) {
+  static getInitialProps({ res, err }) {
     const statusCode = res ? res.statusCode : err ? err.statusCode : null
 
-    // redirect to home now if not found
+    // why did we previously automatically redirect to home?
+    // at any rate, we cannot use Router here because Router doesn't exist server-side
```
Comment (cnasc) [inline unlock-app/src/pages/_error.js]: We should discuss this

Comment (cnasc) [reply unlock-app/src/pages/_error.js]: Maybe we can just adjust that function to be like the example here: https://stackoverflow.com/a/56928111

Comment (julien51) [reply unlock-app/src/pages/_error.js]: I actually don't think it matters. We can just show an error if not found.

Comment (julien51) [reply unlock-app/src/pages/_error.js]: (note that netlify's logic will do that _before_ it hits our app anyway)

**Comment by cnasc** [comment]
🤷‍♂️ another one that runs fine locally. I don't think updating next is worth spending more time figuring out why the integration tests fail.

**Comment by cnasc** [comment]
I think the integration test failures are caused by a weird redirect I've started seeing when I visit `http://locahost:3000`. It immediately tries to redirect me to the static site (`http://locahost:3002`), which may cause some issues when running the integration tests.

**Comment by cnasc** [comment]
Integration tests fail on the dashboard, it looks like the account never populates. I was able to make this run successfully by manually running the test, not sure what makes it fail when running via the script.

**Comment by cnasc** [comment]
Pinning this for now, I'll incrementally update app-by-app so we only have to deal with `unlock-app` in the final PR

**Pull request closed without merging**